i18n/en-US.properties

```diff
@@ -208,6 +208,8 @@ be.deleteDialogFileText = Are you sure you want to delete {name}?
 be.deleteDialogFolderText = Are you sure you want to delete {name} and all its contents?
 # Label for delete confirmation dialog
 be.deleteDialogLabel = Confirm Delete
+# Aria label for button to delete a comment, task, or app activity
+be.deleteLabel = Delete
```

Comment:
Are these changes related to this PR?

Reply:
No, git push automatically put them there

Reply:
I just did a rebase onto the latest from master, so I am not sure why that is happening...

Reply:
Try deleting your `/es` directory, then run `./scripts/prepush.sh`.

Reply:
Yeah, talked with Andrew H and he essentially said the same thing... someone deleted some strings that still existed in my es dir. He recommended to do "yarn clean" and "yarn build" after every rebase. Now the en-US.properties is gone from the PR.

Reply:
to be clear, this can occur if there are stale values in `i18n/json` and a rebased commit contains a deletion to some content referenced there